src/windows/main_window.py

```diff
@@ -649,18 +658,21 @@ def auto_save_project(self):
 
     def actionSaveAs_trigger(self):
         app = get_app()
+        s = app.get_settings()
         _ = app._tr
 
-        recommended_path = app.project.current_filepath
-        if not recommended_path:
-            recommended_path = os.path.join(
-                info.HOME_PATH, "%s.osp" % _("Untitled Project"))
+        recommended_folder = s.getDefaultPath(s.actionType.SAVE)
```

Comment:
This behavior is different now, right? We were using the current_filepath previously, and only using "Untitled Project" when there is no current_filepath. I'm not sure about this though... just curious.

Reply:
Fixed this. Now it'll check for a project name